mud.ttl

```diff
@@ -102,3 +102,12 @@ mud:CharacterList a rdf:Property ;
     rdfs:comment "une liste de personnages appartenant à un compte"@fr ;
     rdfs:domain mud:Account ;
     rdfs:range xsd:string .
+
+mud:content a rdfs:Class ;
+    rdfs:label "Textual Content"@en ;
+    rdfs:comment "Content describing another model"@en ;
+
+mud:sight a rdfs:Property
+    rdfs:label "Sight"@en ;
+    rdfs:comment "The visual sense"@en ;
+    rdfs:domain mud:content
```

Comment:
Could we move the new things into a new ontology please ? I'm thinking `mudcontent` ?

Just for modularity - if someone was writing their own front-end they might want to replace our definitions for the user's perception

Reply:
Done in https://github.com/Multi-User-Domain/vocab/pull/2/commits/1a6f2861f31eea0fef6de765ad814c5708eec396 :smile: 

I then updated the Java app here https://github.com/Multi-User-Domain/mud-jena/pull/7/commits/59a821aff2a2418195fdb902bd7d6dd5b9094885